### DISCUSSION TITLE
Multiple AWS connections support in DynamoDBToS3Operator

### DISCUSSION THREAD

**Discussion by dym-ok** [apache/airflow#29404] [Ideas] [answered]
We're using `DynamoDBToS3Operator` to copy data from a DynamoDB table in one AWS account, where we assume an IAM role, to an S3 bucket in another account.

For that we have to use a slightly modified version of this operator that uses two different AWS connections, with a second one provided as `aws_dynamodb_conn_id` constructor argument.

I think it might be a typical use case and would be willing to provide a PR for such extension. Does anybody find this use case common enough for such a change?

**Comment by Taragolis** [comment, marked as answer]
> I think it might be a typical use case and would be willing to provide a PR for such extension. Does anybody find this use case common enough for such a change?

Feel free to make a PR, that is how Airflow evolve in general:
* User has use-case/issue 
* Make a PR 
* Discussion in the PR (if required)
* PR merged in `main`
* Changes added in new version of specific provider or core

If it would your first PR to Airflow, I suggest to read [Contributor's Quick Start](https://github.com/apache/airflow/blob/main/CONTRIBUTORS_QUICK_START.rst) and all other relevant docs, e.g. [Testing](https://github.com/apache/airflow/blob/main/TESTING.rst), [Static code checks](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#id4)

**Reply by dym-ok** [reply]
Thanks, @Taragolis I'll get onto it.

**Reply by Taragolis** [reply]
One thing it would be nice if changes keep compatibility with current implementation.

In additional might be a good idea to join to the [Apache Airflow Community Slack](https://s.apache.org/airflow-slack), for development questions useful channels are: _development_ and _airflow-breeze_

**Reply by dym-ok** [reply]
Thanks a lot for your tips, @Taragolis!
